### PR TITLE
Update page.mdx

### PR DESCRIPTION
**Pull request by erickirt**
this link leads no where and if spelling is corrected, still leads no where: https://docs.medusajs.com/resources/medusa-container-resources

### CODE REVIEW

**Comment by erickirt** [comment]
Do not merge this, i did not add a fix, just a commented complaint

**Comment by damien-thiesson** [comment]
> Do not merge this, i did not add a fix, just a commented complaint

Hi @erickirt, thank you for your report and contribution. Pull requests are used to fix the source code, rather than just reporting an issue. To report an issue with the documentation without fixing the code yourself, you can click on [Report issue](https://github.com/medusajs/medusa/issues/new?assignees=&labels=type%3A+docs&template=docs.yml&title=Medusa%20v2%20Docs%3A%20Issue%20in%201.%20Introduction%20-%20Medusa%20v2%20Docs) in the top right corner of any documentation page. This will create a [Github issue](https://github.com/medusajs/medusa/issues) that can then be picked up by someone to push an actual PR to fix the link.

Can you close this PR and create an issue instead?

Many thanks :)

**Review by damien-thiesson — CHANGES_REQUESTED** [review]
Requesting to close this PR and create an issue instead.

File: www/apps/book/app/advanced-development/modules/container/page.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ So, resources in the module, such as services or loaders, can only resolve other
 
 ### List of Registered Resources
 
-Find a list of resources or dependencies registered in a module's container in [this Learning Resources reference](!resoures!/medusa-container-resources).
+Find a list of resources or dependencies registered in a module's container in [this Learning Resources reference](!resoures!/medusa-container-resources). <_THIS LEADS NO WHERE
```
Comment (shahednasser) [inline www/apps/book/app/advanced-development/modules/container/page.mdx:17]: Nice catch, can you please change it to this:

```suggestion
Find a list of resources or dependencies registered in a module's container in [this Learning Resources reference](!resources!/medusa-container-resources).
```

File: www/apps/book/app/advanced-development/modules/container/page.mdx [hunk 2]
```diff
@@ -14,7 +14,7 @@ So, resources in the module, such as services or loaders, can only resolve other
...
 
 ---
```
Comment (shahednasser) [inline www/apps/book/app/advanced-development/modules/container/page.mdx:19]: Please don't remove this.

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Pull request closed without merging**